pandas/tests/extension/test_period.py

```diff
@@ -147,7 +148,16 @@ class TestReshaping(BasePeriodTests, base.BaseReshapingTests):
 
 
 class TestSetitem(BasePeriodTests, base.BaseSetitemTests):
-    pass
+
+    def test_setitem_slice_mismatch_length_raises(self, data):
```

Comment:
Ahh, just noticed. This should be on the base class BaseSetitemTests. Then all the subclasses will be tested, including period.

Reply:
Same for the other test.

Reply:
ahh, i see, yeah, makes sense!! thx!